### ISSUE TITLE
[A11y][Contrast]Color contrast for 'Cancel' button foreground with background is 1.099:1 which is less than 3:1

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8192]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188116
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450) 

## Note:
Same issue is observed for throughout the application on 'Cancel" button
## Repro Steps:

    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "API Keys" and Press enter
    Tab till "Create button" and press enter
    Tab till "Cancel button" under the 'Select packages'  
    Check whether color contrast for 'Cancel' button's foreground with page background is meeting the required 3:1 ratio or not 

## Actual Results:

Color contrast for 'Cancel' button foreground with background is 1.099:1 which is less than 3:1
## Expected Results:
Low vision user  should be able to identify the button, so it should meet the required color contrast ratio 3:1
## User Impact:
Low vision users may not be able to identify the button if it is not meeting the required color contrast ratio.

## Notes
Looks like this will need a global fix. I believe we use a single set of CSS for the cancel button, so it should be sufficient to change this one time.

**Comment by loic-sharma** [comment]
Verify:

* [ ] Cancel create API key form
* [ ] Packages
  * [ ] Cancel package upload on verify page
  * [ ] Accept new package owner
  * [ ] Reject new package owner
* [ ] Account
  * [ ] Sign in using external account not linked to a nuget.org account
  * [ ] Register using external account
  * [ ] Sign in using external account
  * [ ] Link nuget.org account to external account (prompted after login)
  * [ ] Link nuget.org account to different external account on profile page
  * [ ] Cancel "Why did you choose not to enable 2FA?" survey
  * [ ] Enable 2FA on profile page
  * [ ] Transform to organization
  * [ ] Cancel transform to organization form
  * [ ] Cancel pending email change
  * [ ] Cancel email change form
  * [ ] Cancel change notifications form
  * [ ] Cancel change password form
  * [ ] Cancel link to external account form
  * [ ] Cancel delete account
* [ ] Organizations
  * [ ] Cancel creating organization
  * [ ] Cancel deleting organization on organization profile page
  * [ ] Cancel deleting organization on confirm deletion page
  * [ ] Accept organization membership
  * [ ] Reject organization membership
* [ ] Admin
  * [ ] Cancel editing a flight
